### PR TITLE
feat: event search on new endpoint, first test

### PR DESCRIPTION
**Pull request by sjaanus**
Changed the url of event search to search/events to align with search/features. With that created a search controller to keep all searches under there.
Added first test.

### CODE REVIEW

File: src/lib/routes/admin-api/event.ts [hunk 1]
```diff
@@ -125,26 +115,6 @@ export default class EventController extends Controller {
                 }),
             ],
         });
-
```
Comment (sjaanus) [inline src/lib/routes/admin-api/event.ts:128]: This is just moved to separate controller.

File: src/lib/features/events/event-store.ts [hunk 2]
```diff
@@ -89,8 +89,8 @@ export interface IEventTable {
 
 export interface IEventSearchParams {
     query: string | undefined;
-    offset: string | undefined;
-    limit: string | undefined;
+    offset: number | undefined;
+    limit: number | undefined;
```
Comment (thomasheartman) [inline src/lib/features/events/event-store.ts:93]: Why do we have two interfaces with the exact same name now? Did we always? Is it gonna cause confusion?

File: src/lib/features/events/event-service.ts [hunk 3]
```diff
@@ -8,10 +8,21 @@ import type EventEmitter from 'events';
...
+    project?: string;
+    query?: string;
+    createdAtFrom?: string;
+    createdAtTo?: string;
+    createdBy?: string;
+    type?: string;
+    offset: number;
+    limit: number;
+    // sortBy?: string;
+}
```
Comment (thomasheartman) [inline src/lib/features/events/event-service.ts:24]: Does the openapi schema not cut it anymore?

**Review by thomasheartman — APPROVED** [review]
Nice! Just a little skeptical of using the same name for two interfaces that where one is a subset of the other, but I'll let you be the judge of that 💁🏼

File: src/lib/features/events/event-store.ts (hunk 2 above)
Comment (sjaanus) [reply src/lib/features/events/event-store.ts:93]: Good catch, removed one.

**Pull request merged**